### PR TITLE
Add missing attachments & publish pdf to wiki

### PR DESCRIPTION
**Pull request by maximthomas**

### CODE REVIEW

File: openidm-doc/src/main/asciidoc/attachments/provisioner.openicf-adldap.json [hunk 1]
```diff
@@ -0,0 +1,681 @@
+{
+  "name": "AD",
+  "connectorRef": {
+    "connectorHostRef": "#LOCAL",
+    "connectorName": "org.identityconnectors.ldap.LdapConnector",
+    "bundleName": "org.forgerock.openicf.connectors.ldap-connector",
+    "bundleVersion": "1.4.1.0"
```
Comment (vharseko) [inline openidm-doc/src/main/asciidoc/attachments/provisioner.openicf-adldap.json:7]: set bundle version as range https://github.com/OpenIdentityPlatform/OpenIDM/blob/1226eb305362efe8b61bfe02cb008ad5a49815c1/openidm-zip/src/main/resources/samples/provisioners/provisioner.openicf-adldap.json#L7C9-L7C40

File: openidm-doc/src/main/asciidoc/attachments/provisioner.openicf-adldap.json [hunk 2]
```diff
@@ -0,0 +1,681 @@
+{
+  "name": "AD",
+  "connectorRef": {
+    "connectorHostRef": "#LOCAL",
+    "connectorName": "org.identityconnectors.ldap.LdapConnector",
+    "bundleName": "org.forgerock.openicf.connectors.ldap-connector",
```
Comment (vharseko) [inline openidm-doc/src/main/asciidoc/attachments/provisioner.openicf-adldap.json:6]: "bundleName" : "org.openidentityplatform.openicf.connectors.ldap-connector",

File: openidm-doc/src/main/asciidoc/attachments/provisioner.openicf-adldap.json (hunk 1 above)
Comment (maximthomas) [reply openidm-doc/src/main/asciidoc/attachments/provisioner.openicf-adldap.json]: done 2517a93

File: openidm-doc/src/main/asciidoc/attachments/provisioner.openicf-adldap.json (hunk 2 above)
Comment (maximthomas) [reply openidm-doc/src/main/asciidoc/attachments/provisioner.openicf-adldap.json]: done 2517a93

**Review by vharseko — APPROVED** [review]

**Pull request merged**